expo/tests/gbe/test_review_profiles_view.py

```diff
@@ -42,6 +43,14 @@ def test_all_well(self):
         login_as(self.privileged_user, self)
         response = self.client.get(self.url)
         self.assertEqual(200, response.status_code)
+        self.assertContains(response, "Manage Users")
+
+    def test_staff_lead(self):
+        context = StaffAreaContext()
+        login_as(context.staff_lead, self)
+        response = self.client.get(self.url)
+        self.assertEqual(200, response.status_code)
+        self.assertContains(response, "Manage Users")
```

Comment:
Let's add one to test that OLD staff leads (past conferences) DON'T get this privilege.